migrations/20180418101950_create-transfer-state-change.js

```diff
@@ -5,11 +5,8 @@ exports.up = function (knex, Promise) {
     t.bigIncrements('transferStateChangeId').primary().notNullable()
 
     t.string('transferId', 36).notNullable()
+    t.foreign('transferId').references('transferId').inTable('transfer')
```

Comment:
Why has this been added as per references the commented out code below?

Reply:
They were renamed because the previous order did not work for adding a reference to the transfer table. The commented out code below actually has not been implemented in other migration script.